tools/downloader/pytorch_to_onnx.py

```diff
@@ -24,6 +24,13 @@ def positive_int_arg(values):
     return result
 
 
+def prepare_model_params(module, model_params):
+    for key in model_params:
+        func = model_params[key].split(".", 1)[0]
+        if hasattr(module, func):
+            model_params[key] = eval("module." + model_params[key])
```

Comment:
could you please describe for what do you use parameters preparation and why simple eval is not enough in your case

Reply:
because function to construct model parameters with simple eval is called before inclusion necessary module.

Reply:
can we just move inclusion necessary modules before parameters evaluation?
If not, probably this situation should be detected automatically, on my view, without additional user parameters specification. The reason is user is not obligated to know internal mechanics of argument parsing, so now it looks like workaround (if usual loading does not work, then enable this parameter)

Reply:
Delete this additional parameter and add eval to postprocessing in model.yml